### ISSUE TITLE
Cyclops drop

### ISSUE THREAD

**Issue by zEpiixz** [AlexModGuy/Ice-and-Fire-Suggestions#285]
i think the cyclops isnt really rewarding. So most of the time you dont fight him because of this.
So hows about a nice drop for him. Maybe smth like his eye and with that eye in hand you could see sheeps in a radious or smth. Just something that fighting one would be worth.

**Comment by samoja12** [comment]
Yeah, even if it does not have a purpose for the time being a trophy would be nice, i like to collect trophies and display them in my throne room in my castle, i have fire and frost dragon hearts, stymphalian bird feather and death worm scale on display, i plan to get troll tooth, shiny scales and gorgon head next, but cyclops does not drop anything i could display as a trophy.
